### PR TITLE
Fix edition when node/process read only

### LINKED ISSUE #394: Node details modal is editable in archived processes
Edition should be disabled

### LINKED ISSUE #395: Frontend control for mapVariable does not respect readOnly
E.g. in compare modal, buttons for adding/removing fields in mapVariable are enabled

### PR DESCRIPTION
**Pull request by PPiedel**
This fixes #395 
This fixes #394
This fixes disabled node opacity

### CODE REVIEW

**Comment by witekw** [comment]
#395 - Can you hide "-" "+" buttons?

image

**Comment by PPiedel** [comment]
> #395 - Can you hide "-" "+" buttons?
> 
> image

Done

File: ui/client/components/Fields.js [hunk 1]
```diff
@@ -143,12 +150,18 @@ export default class Fields extends React.Component {
                  removeField={this.removeField.bind(this)}
                  moveItem={moveItem}
                  showValidation={this.props.showValidation}
```
Comment (lciolecki) [inline ui/client/components/Fields.js]: Myaby better will be extract all props data?

Comment (PPiedel) [reply ui/client/components/Fields.js]: Done

**Pull request merged**